docs/docs/bootstrapping.md

```diff
@@ -0,0 +1,61 @@
+<div class="content-title">Bootstrapping</div>
+
+***
+
+### Using the maxwell-bootstrap utility
+
+You can use the `maxwell-bootstrap` utility to bootstrap tables from the command-line.
+
+option                                        | description
+--------------------------------------------- | -----------
+--log_level LOG_LEVEL                         | log level (DEBUG, INFO, WARN or ERROR)
+--user USER                                   | mysql username
+--password PASSWORD                           | mysql password
+--host HOST                                   | mysql host
+--port PORT                                   | mysql port
+--database DATABASE                           | mysql database containing the table to bootstrap
+--table TABLE                                 | mysql table to bootstrap
+
+### Using the maxwell.bootstrap table
+
+Alternatively you can insert a row in the `maxwell-bootstrap` table to trigger a bootstrap.
+
+```
+mysql> insert into maxwell.bootstrap (database_name, table_name) values ('fooDB', 'barTable');
+```
+
+### Async vs Sync bootstrapping
+
+The Maxwell replicator is single threaded; events are captured by one thread from the binlog and replicated to Kafka one message at a time.
+When running Maxwell with `--bootstrapper=sync`, the same thread is used to do bootstrapping, meaning that all binlog events are blocked until bootstrapping is complete.
```

Comment:
open-replicator feeds into a LinkedBlockingQueue of a few hundred events, and will just start blocking if maxwell isn't there to get them.  no biggie.